README.md

```diff
@@ -264,6 +264,26 @@ this command is used to run [graphql code generator](https://graphql-code-genera
 
 4. you should run `npm run generate` every time new unions or interfaces are created
 
+### Implementing Facebook chat.
+
+For run Facebook chat on your site you need two variables - FACEBOOK_PAGE_ID and FACEBOOK_APP_ID and write them
+in '.env' file.
+
+1. FACEBOOK_PAGE_ID.
+ You can find your page ID in menu under your avatar, button 'about' or 'more -> about'.
+ Then scroll to the down and find your Page ID.
+ 
+2. FACEBOOK_APP_ID
+ Go to https://developers.facebook.com/apps/ and choose 'Add a New App -> Manage Business Integration'.
+ Fills the forms and press 'Create App ID'. You can find your App ID at the top of page.
+ 
+3. After that you may go to left bar, and find 'Messenger -> Settings' and add your Facebook Business page ID in section 
+ 'Access Tokens'.
+
+4. Now go to your Facebook Business page -> Settings(Left bar) -> Advanced Messaging and find section 'Whitelisted 
+ Domains'. Here you must add your domain to white list and save.
+ 
```

Comment:
Please, add somewhere here the link to the official tutorial from facebook how to setup Facebook messenger for more concrete information

Reply:
Done